### PR TITLE
Fix issues of NuGet arthoring projects in FxCop analyzers

### PR DESCRIPTION
**Pull request by genlu**
- Make content of existing nuspec files identical to their counterpart in dotnet/roslyn repo.
- Add Nuget authoring to Desktop.Analyzers.

@tmeschter @srivatsn @mavasani @lgolding

### CODE REVIEW

File: src/FxCop/Desktop.Analyzers/Core/DesktopAnalyzers.nuspec [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Desktop.Analyzers</id>
+    <summary>Desktop Analyzers</summary>
+    <description>
+      Analyzers for the APIs only exist in .NET Framework.
```
Comment (tmeschter) [inline src/FxCop/Desktop.Analyzers/Core/DesktopAnalyzers.nuspec]: "Analyzers for APIs specific to the desktop .NET Framework."

File: src/Packaging/Packaging.proj [hunk 2]
```diff
@@ -6,6 +6,7 @@
   <ItemGroup>
     <Project Include="..\MetaCompilation\MetaCompilation\MetaCompilation.Nuget\MetaCompilation.Nuget.proj" />
     <Project Include="..\CodeAnalysis\NuGet\CodeAnalysisDiagnosticAnalyzers.NuGet.proj" />
+    <Project Include="..\FxCop\Desktop.Analyzers\NuGet\DEsktopAnalyzers.NuGet.proj" />
```
Comment (tmeschter) [inline src/Packaging/Packaging.proj]: Minor: check the casing: DEsktop -> Desktop.

**Comment by tmeschter** [comment]
:+1:

**Comment by srivatsn** [comment]
:+1:

**Comment by genlu** [comment]
@dotnet-bot Retest this, please.

**Pull request merged**